src/Tests/Microsoft.DotNet.GenAPI.Tests/CSharpFileBuilderTests.cs

```diff
@@ -1733,7 +1733,7 @@ namespace A
                     {
                         public partial class foo
                         {
-                            public void Execute(out int i) { throw null; }
+                            public void Execute(scoped out int i) { throw null; }
```

Comment:
This looks like a bug to me. Why would the ref API have `scoped out int` when the implementation has `out int`?